.pylintrc

```diff
@@ -1,6 +1,23 @@
 [MESSAGES CONTROL]
 
 disable=
-  unused-wildcard-import,
+  # Ignore refactor and convention categories
   R,
-  C
+  C,
+  # Allow TODO and FIXME comments
+  fixme,
+  # Ignore unnecessary-lambda because it has many false positives from hl.ArrayExpression.map, etc.
+  unnecessary-lambda,
+  # Warnings present in existing code. These should be fixed and removed from this list.
```

Comment:
So is this something we should fix first and then remove these lines? We can put in this PR, but I am also happy to try to fix these after. I'm assuming I will just see them all when I run ./lint?

Reply:
Yes, the code causing the warning will have to be fixed before the warning can be removed from this list. Otherwise, PR checks will fail.

`./lint` will now succeed because all of these warnings are disabled. To enable one and find code that causes it, either remove it from this list and run `./lint` or enable it it Pylint CLI args (ex: `./lint --enable=unused-import`).